modules/oe_webtools_maps/oe_webtools_maps.routing.yml

```
@@ -0,0 +1,7 @@
+oe_webtools_maps.settings:
+  path: '/admin/config/system/oe_webtools_maps'
+  defaults:
+    _form: 'Drupal\oe_webtools_maps\Form\WebtoolsMapsSettingsForm'
+    _title: 'Webtools Maps Form settings'
```

```suggestion
    _title: 'Webtools Maps settings'
```
Drop the form a bit everywhere :)